test/unit/extension.test.ts

```diff
@@ -103,10 +103,16 @@ suite('openshift toolkit Extension', () => {
 	});
 
     test('should register all extension commands declared commands in package descriptor', async function() {
+        const notRegisteredCommands = [];
         const commands = await vscode.commands.getCommands(true);
         packagejson.contributes.commands.forEach((value)=> {
-            expect(commands.includes(value.command), `Command '${value.command}' handler is not registered during activation`).true;
+            try {
```

Comment:
instead if a try/catch, you can use an if:

```typescript
if (!commands.includes(value.command)) {
  notRegisteredCommands.push(value.command);
}
```

Reply:
ACK